src/features/student/declaredPrograms/components/overlays/AddDeclaredProgramDrawer/AddDeclaredProgramDrawer.vue

```diff
@@ -0,0 +1,132 @@
+<script setup lang="ts">
+import { ConfirmationModal } from '@/common/components'
+import { useModal } from '@/common/composables'
+import DeclaredProgramDescriptionFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramDescriptionFormField/DeclaredProgramDescriptionFormField.vue'
+import DeclaredProgramInstitutionFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramInstitutionFormField/DeclaredProgramInstitutionFormField.vue'
+import DeclaredProgramLinkFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramLinkFormField/DeclaredProgramLinkFormField.vue'
+import DeclaredProgramPeriodFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramPeriodFormField/DeclaredProgramPeriodFormField.vue'
+import DeclaredProgramResultFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramResultFormField/DeclaredProgramResultFormField.vue'
+import DeclaredProgramSourceOfInformationFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramSourceOfInformationFormField/DeclaredProgramSourceOfInformationFormField.vue'
+import DeclaredProgramTitleFormField from '@/features/student/declaredPrograms/components/interactions/formFields/DeclaredProgramTitleFormField/DeclaredProgramTitleFormField.vue'
+import { useAddDeclaredProgramForm } from '@/features/student/declaredPrograms/components/overlays/AddDeclaredProgramDrawer/use-add-declared-program-form/use-add-declared-program-form'
+import { useDeclaredProgramsStore } from '@/features/student/declaredPrograms/stores/declaredPrograms.store'
+import { useToasterStore } from '@/store'
+import { AvAccordion, AvAccordionsGroup, AvCancelConfirmButtons, AvDrawer, MDI_ICONS } from '@avenirs-esr/avenirs-dsav'
+import { useI18n } from 'vue-i18n'
+
+const { t } = useI18n()
+const declaredProgramsStore = useDeclaredProgramsStore()
+const { addSuccessMessage } = useToasterStore()
+const showDrawer = toRef(declaredProgramsStore, 'showAddDeclaredProgramDrawer')
+const activeAccordion = ref(0)
```

Comment:
```suggestion
const activeAccordion = ref<DeclaredProgramAccordions>(DeclaredProgramAccordions.ADD_PROGRAM)
```

Avec

```js
enum DeclaredProgramAccordions {
  ADD_PROGRAM = 0,
  // ...
}
```

Tu nous avais fait faire ce genre de change, c'est pour ça que je le demande ici :+1:

Reply:
@p-beger on l'a jamais fait pour un accordion c'est plutot sur Tabs pour identifier la actif
la j'ai pas besoin carrement, j'ai viré le v-model